PACKAGES.md

```diff
@@ -104,7 +104,7 @@ Placing an executable script beneath `/srv/scripts/$PACKAGE_NAME/run`
 creates a _service_ – a long-running supervised process that is
 automatically restarted upon termination, and can be controlled via the
 set of management scripts described in
-[INTERNALS.md](https://github.com/medic/medic-os/blob/master/INTERNALS.md).
```

Comment:
@Hareet, @henokgetachew, if you look at this URL (even the old version without my change to main), it goes to a broken page. Do you know where it should go so I can fix it here?

Reply:
I'm not sure. We'd have to look in the git history to look for that file. I suspect it's been renamed to a README file.